includes/admin/class-amp-post-meta-box.php

```diff
@@ -292,6 +292,40 @@ function () {
 		}
 	}
 
+	/**
+	 * Returns the featured image dimensions after filtering
+	 * the height and width
+	 *
+	 * @return array
+	 */
+	public static function get_featured_image_dimensions() {
+		return [
+			/**
+			 * Filter name: amp_featured_image_minimum_height
+			 *
+			 * @since 2.0.8
+			 *
+			 * @param int $featured_image_minimum_height The minimum height of the image,
+			 * defaults to 675
+			 *
+			 * @return int
```

Comment:
```suggestion
```

Filter phpdoc does not include `@return`. See https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-hooks-actions-and-filters